packages/investor-idle/src/IdleSdk.ts

```diff
@@ -1,12 +1,15 @@
-// import https from 'https'
-import { availableTokens, IdleVault } from './constants'
+import axios from 'axios'
+
+import { IdleVault } from './constants'
 
 export class IdleSdk {
   async getVaults(): Promise<IdleVault[]> {
-    // const vaults = await makeRequest('http://localhost:3333/pools?api-key=bPrtC2bfnAvapyXLgdvzVzW8u8igKv6E');
-    // if (vaults){
-    // this.#vaults = JSON.parse(vaults);
-    // }
-    return availableTokens
+    const results = await axios.get(
+      'https://api.idle.finance/pools?api-key=bPrtC2bfnAvapyXLgdvzVzW8u8igKv6E'
+    )
+    if (results && results.data) {
+      return results.data.filter((d: IdleVault) => !!d.externalIntegration)
```

Comment:
Non-blocking but we tent to use full item names when iterating, which in this case would be
 ```suggestion
      return results.data.filter((data: IdleVault) => !!data.externalIntegration)
```